app/services/sinatra_service.rb

```diff
@@ -31,4 +35,9 @@ def get_json(url, params = nil)
     response = conn.get(url, params)
     JSON.parse(response.body, symbolize_names: true)
   end
+  
+  def get_json_not_symbolized(url, params = nil)
+    response = conn.get(url, params)
+    JSON.parse(response.body)
+  end
```

Comment:
Let's please leave this code in here. It's necessary for our member_votes endpoint as we need the keys returned as strings and not symbols